ext/date/php_date.stub.php

```diff
@@ -692,6 +693,26 @@ public function __wakeup(): void {}
     public static function __set_state(array $array): DateInterval {}
 }
 
+final class DateTimeStringInterval extends DateInterval
+{
+    public readonly ?string $date_string;
+
+    public function __construct(string $datetime) {}
+}
+
+final class DateTimeInterval extends DateInterval
+{
+    public readonly int $y;
+    public readonly int $m;
+    public readonly int $d;
+    public readonly int $h;
+    public readonly int $i;
+    public readonly int $s;
+    public readonly float $f;
+    public readonly int $invert;
+    public readonly int|false $days;
```

Comment:
IMHO -1 for false would be better than mixed types

Reply:
I disagree here.